python/setup.py

```diff
@@ -2,28 +2,30 @@
 from version import get_git_version
 
 setup_args = dict(
-  name = 'swiftnav',
-  version = get_git_version(),
-  description = 'Python bindings to the libswiftnav library.',
-  license = 'LGPLv3',
-  url = 'http://www.swiftnav.com',
-  author = 'Swift Navigation Inc.',
-  author_email = 'dev@swiftnav.com',
-  maintainer = 'Swift Navigation',
-  maintainer_email = 'dev@swiftnav.com',
-  packages = ['swiftnav'],
+    name='swiftnav',
```

Comment:
Hmm, I will ask... @benjamin0 Do you know what the story is with our Python formatting?